bin/local-env/docker-compose.yml

```
@@ -2,7 +2,7 @@ version: '3.1'
 
 services:
   wordpress:
-    image: wordpress
+    image: wordpress:php${PHP_VERSION}
```

```suggestion
    image: wordpress:php${PHP_VERSION:=7.4}
```

This will use 7.4 as the default if it's not specified

---

I was just googling how to do this!